src/components/Button/style.scss

```diff
@@ -0,0 +1,33 @@
+/* Import Variables and Helpers */
+@import '../../style/helpers';
+
+/* Local Variables */
+$BUTTON_HEIGHT: 60 !default;
+$BUTTON_WIDTH: 280 !default;
+$BUTTON_RADIUS: 5 !default;
+
+.button {
+  display: block;
+  margin: 0 auto;
+  width: 100%;
+  max-width: remcalc($BUTTON_WIDTH);
+  height: remcalc($BUTTON_HEIGHT);
+  line-height: remcalc($BUTTON_HEIGHT);
```

Comment:
Buttons shouldn't have max-width, height or line-height. it should be simple container with paddings and e.g. display flex to easily center children or create space between them with justify content

Reply:
We'll discuss this tomorrow because I'm pretty sure there will be padding issues on different browsers. That's being caused by different character length and in some browsers we would need to fix that padding with extra +/- 1px to make them look centered, but as I told you, We can discuss that matter tomorrow. About the width, I used to do fluid buttons as base for ages with modifiers if we need to make them smaller etc but in this case All the buttons got static width and I think they should remain like that because on higher resolutions with wider container they would look like big blue bars.